src/containers/genePage/index.js

```diff
@@ -127,6 +131,14 @@ const GenePage = ({geneId}) => {
           />
         </Subsection>
 
+        <Subsection help={<PathwayUserGuide />} t title={PATHWAY}>
```

Comment:
the `t` prop in the Subsection component might have been a type?

Reply:
Yep it's a typo :) Thanks for spotting it